module/VuFind/src/VuFind/Db/Service/PluginManager.php

```diff
@@ -29,6 +29,11 @@
 
 namespace VuFind\Db\Service;
 
+use VuFind\Db\Table\AccessToken;
+use VuFind\Db\Table\GatewayFactory;
+use VuFind\Db\Table\User;
```

Comment:
As noted elsewhere, I think it would be better to create actual service classes instead of just applying interfaces to the table classes, even if all the services do for now is accept a table object through the constructor (or use the DbTableAwareInterface) and proxy appropriate methods. I feel this way for a few reasons:

1.) If we start to establish a parallel structure between dev and #2233, it will make reviewing and understanding that PR a lot easier.

2.) This ensures that we build the service classes and interfaces thoughtfully, only adding the methods that we need and want. If we just apply interfaces to table classes, we might end up accidentally relying on methods that are not part of the interface, which could cause problems later. (I realize that static analysis could help with this to some extent, but I don't think the typing in our code is robust enough to guarantee accurate analysis at this point).

3.) If we need to do mapping between the native return values provided by Laminas\Db and the standard interfaces we define, the service class would be a good place to do this if we want to retain the legacy table methods for backward compatibility and/or transitional purposes (e.g. if we want our service to return an array, but the table returns a Laminas ResultSet).